tests/testthat/test-int-latent_individual.R

```diff
@@ -47,7 +47,7 @@ test_that("epidist.epidist_latent_individual samples from the prior according to
   epidist_formula <- epidist_formula(
     data = prep_obs,
     family = epidist_family,
-    formula = brms::bf(mu ~ 1, sigma ~ 1)
+    formula = mu ~ 1
```

Comment:
do we need a bf test now or are we still covered?

Reply:
Test that `epidist_formula` produces same thing with `bf`

Reply:
[136f012](https://github.com/epinowcast/epidist/pull/396/commits/136f0128045baf796aa3367dfe61b723c0d7afe3)